entrypoint.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
 cd /home/$USER/ansible-obs
-bundle.ruby3.1 config build.nokogiri --use-system-libraries && bundle.ruby3.1 install --deployment --binstubs
+bundle.ruby3.1 config set --local deployment 'true' &&
+bundle.ruby3.1 config build.nokogiri --use-system-libraries &&
+bundle.ruby3.1 install &&
+bundle.ruby3.1 binstubs --all
```

Comment:
Just as a note for any potential onlookers in the future, this change was to address another deprecation warning:

```
[DEPRECATED] The --binstubs option will be removed in favor of `bundle binstubs --all`
```

So `bundle.ruby3.1 install --binstubs` became `bundle.ruby3.1 binstubs --all`.